tests/testImageDecorrelation.py

```diff
@@ -54,25 +55,26 @@ def singleGaussian2d(x, y, xc, yc, sigma_x=1., sigma_y=1., theta=0.):
     return out
 
 
-def makeFakeImages(xim=None, yim=None, sig1=0.2, sig2=0.2, psf1=2.2, psf2=3.3, offset=None,
-                   psf_yvary_factor=0., varSourceChange=1/50., theta1=0., theta2=0., im2background=0.,
+def makeFakeImages(xim=None, yim=None, var1=0.04, var2=0.04, psf1=3.3, psf2=2.2, offset=None,
+                   psf_yvary_factor=0., varSourceChange=1/50., theta1=0., theta2=0., im1background=0.,
                    n_sources=500, seed=66, verbose=False):
     """! Make two exposures: a template and a science exposure.
     Add random sources of identical flux, with randomly-distributed fluxes and a given PSF, then add noise.
+    In all cases below, index (1) is the science image, and (2) is the template.
```

Comment:
I'd suggest using "science" and "template" everywhere in the code to make that explicit. They're not symmetric, so don't treat them like they are.